test/end-to-end/README.md

```diff
@@ -105,18 +105,18 @@ describe('View Recipe Page', function () {                // Which page does thi
     after((done) => {                                     // Scenario de-provision, like apply a snapshot
     });
 
-    context('Menu Nav Bar Check', () => {                 // Element or elements to be test
-      it('should show a recipe name', (done) => {         // Expected result description
+    context('Menu Nav Bar Check #Acceptance', () => {     // Test content and test type
+      it('should show a recipe name @Welder-Web-40003', (done) => {    // Case description and case ID
```

Comment:
Sorry for confused you. Let me explain the use case of case ID.
The case ID number is based on page:
* 1xxxx - Edit Recipe Page
* 2xxxx - Create Recipe Page
* 3xxxx - Recipes Page
* 4xxxx - View Recipe Page
If only Recipes Page cases need to be run, "mocha --grep @Welder-Web-3*" command will be executed.